cypress/e2e/upload-download.cy.ts

```diff
@@ -1,10 +1,17 @@
-import {Upload} from "../page/index";
+import {
+  Upload,
+  Download,
 
-describe("uploading a file test", () => {
+}
+  from "../page/index";
```

Comment:
Sorry @JulianO911, I gave you a wrong suggestion. Correct modification is this:
```suggestion
} from "../page/index";
```